flycheck.el

```diff
@@ -5605,6 +5605,16 @@ Relative paths are relative to the file being checked."
   :safe #'flycheck-string-list-p
   :package-version '(flycheck . "0.18"))
 
+(flycheck-def-option-var flycheck-dmd-flags nil d-dmd
+  "A list of compiler flags for dmd.
+
+The value of this variable is a list of strings, where each
+string is a compiler flag to add when calling dmd.
+Relative paths are relative to the file being checked."
+  :type '(repeat (string :tag "Compiler flag"))
+  :safe #'flycheck-string-list-p
+  :package-version '(flycheck . "0.23"))
```

Comment:
Please fix the version number.  0.23 is already released, so this option will be introduced in Flycheck 0.24.